src/main/java/com/minecolonies/coremod/proxy/ServerProxy.java

```diff
@@ -1,4 +1,4 @@
-package com.minecolonies.proxy;
+package com.minecolonies.coremod.proxy;
 
 public class ServerProxy extends CommonProxy
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)